src/robot_dart/control/manifold_control.hpp

```diff
@@ -0,0 +1,25 @@
+#include <robot_dart/control/pd_control.hpp>
+
+namespace robot_dart {
+    namespace control {
+
+        class ManifoldControl: public PDControl {
+        public:
+
+            ManifoldControl(const std::vector<int>& t_rotational_dofs );
```

Comment:
Accepting a vector of `int` is inconsistent with the rest of the libraries that accepts a vector of `std::string` (for DOF names).

Reply:
I will deduce internally the type of joint as you suggested